driver-sync/src/test/functional/com/mongodb/client/AbstractUnifiedTest.java

```diff
@@ -77,7 +79,7 @@
 import static org.junit.Assume.assumeTrue;
 
 @RunWith(Parameterized.class)
-public abstract class AbstractTransactionsTest {
+public abstract class AbstractUnifiedTest {
```

Comment:
Since localizing the operations in the subclasses seems awkward, I decided to get a head start on unifying the tests. I have renamed the class to reflect this.

Reply:
Ack